### DISCUSSION TITLE
Rust Plugin Rollback Action

### DISCUSSION THREAD

**Discussion by dagbs** [gameserverapp/Platform#493] [Suggestions]
There is a plugin called [Image Library](https://umod.org/plugins/image-library) which released an update with the description of:
```
This is for the coming Rust update
Do not update before the game update
```

I was able to log into FTP and manually upload the previous version but would have been nice to have that as an option in the panel. I don't expect to be able to select any version, although that would be nice, I would just like the option to keep at least the previous version as a backup to use.

**Comment by mvaessen** [comment]
hi, while i can see this being useful, I don't think it's GSA's job to work around improper versioning. Other plugin sites use special `unstable` flags which GSA automatically ignores. I recommend reaching out to the source of the problem for a fix.
